Quantum-Control-Applications-QuAM/Superconducting/calibrations/00_hello_qua.py

```diff
@@ -1,20 +1,19 @@
 """
 A simple sandbox to showcase different QUA functionalities during the installation.
 """
-
+import os
 from qm.qua import *
 from qm import SimulationConfig
-from components import QuAM
-from macros import node_save
+from quam_components import QuAM
 
 ###################################################
 #  Load QuAM and open Communication with the QOP  #
 ###################################################
 # Instantiate the QuAM class from the state file
-machine = QuAM.load("state.json")
+machine = QuAM.load(os.path.join('..', 'configuration', 'quam_state'))
```

Comment:
We need a better solution for this, two issues I'm seeing:
- The code is difficult to read
- The actual path depends on the current working directory.

We should discuss alternatives

Reply:
Good catch, I agree that the actual path should be relative to the script, not the CWD. I have changed this, but I don't know if the readability can be improved. Thoughts on:

```python
# Get the directory of the current script
current_dir = os.path.dirname(os.path.abspath(__file__))
# Define a path relative to this script, i.e., ../configuration/quam_state
config_path = os.path.join(current_dir, '..', 'configuration', 'quam_state')
# Load the machine state
machine = QuAM.load(config_path)
```
?

Reply:
I'm looking for a solution where we don't need to do any path manipulation in a script, but maybe we do this for a follow-up PR.
For now, I suggest we use pathlib which should look a bit nicer:
```
from pathlib import Path
config_path = Path(__file__).parent.parent / "configuration" / "quam_state"
```

Reply:
I agree.

I have implemented your suggestion for now.